site/_data/versions.yml

```diff
@@ -1,4 +1,4 @@
 vega: 5.21.0
 vega-lite: 5.1.1
-vega-embed: 6.19.1
+vega-embed: 6.20.0
```

Comment:
I have no idea why this bumped when I ran `yarn install && yarn test`

Reply:
This happens when you run `yarn build:versions`. We usually run it before releases.

Reply:
ah, okay, I'll revert this file then.